### PR TITLE
Blog post: The real weakest link in software supply chain security

### PR DESCRIPTION
**Pull request by rebeccadee**
PREVIEW: https://deploy-preview-5205--sourcegraph.netlify.app/blog/real-weakest-link-in-software-supply-chain-security/

https://app.asana.com/0/1199959077348208/1200076936776936/f

### CODE REVIEW

File: blogposts/2022/third-party-vulnerabilties-process.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
+---
+title: 'The real weakest link in software supply chain security (it’s not open source)'
+description: 'When a critical security vulnerability is identified, your response time is everything. There are probably shortcomings in your response process itself that are slowing you down—find out where they are and how you can be better prepared for the next third-party vulnerability.'
+author: Rebecca Dodd, André Eleuterio
+authorUrl: https://handbook.sourcegraph.com/team/#andr%C3%A9-eleuterio
+publishDate: 2022-03-17T18:00+02:00
+tags: [blog]
+slug: real-weakest-link-in-software-supply-chain-security
+heroImage:
+socialImage:
```
Comment (rebeccadee) [inline blogposts/2022/third-party-vulnerabilties-process.md:10]: TODO

File: blogposts/2022/third-party-vulnerabilties-process.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+Again, the teams tasked with mitigation will need to watch out for those pesky transitive dependencies. This might require combing through an open source project’s repository in addition to your own.
+
+### How to fix it
+
+The solutions here will be largely the same as those for #1, with thorough code search playing a key role in sweeping your codebase or that of your dependencies for instances of the vulnerable code. [Nutanix](https://www.nutanix.com/) used Sourcegraph for its Log4Shell management process: “Isn’t it nice, when you can just run a report and say, ‘Here it is’ or ‘Here it isn’t’?” said Jon Kohler, Technical Director of Solution Engineering. Much better than having to say “‘Well, boss, I think we got it all.’”
```
Comment (rebeccadee) [inline blogposts/2022/third-party-vulnerabilties-process.md:36]: TODO: Add link to case study

File: blogposts/2022/third-party-vulnerabilties-process.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+### How to fix it
+
+You can proactively reduce your dependencies by committing to a culture of code reuse in your engineering organization. Code reuse doesn’t just narrow the scope of your dependencies, but reduces time spent on writing duplicative code and streamlines maintenance of shared code. Code search can help engineers discover reliable, reusable code already in your codebase. Using code notebooks to document how and where code is used in your organization also makes it easier for developers to reuse shared libraries confidently, reducing your overall dependency burden.
+
+_TODO: Add summary with link to fixing vulnerabilities with Nutanix webinar when landing page is live_
```
Comment (rebeccadee) [inline blogposts/2022/third-party-vulnerabilties-process.md:56]: TODO

**Comment by rebeccadee** [comment]
@nickmyyz I will pop in tomorrow to add the images and resolve outstanding todos, but would be super helpful to get an approving review from you in the meantime so I'm not blocked from merging tomorrow AM! 🙏 TY!

File: blogposts/2022/third-party-vulnerabilties-process.md (hunk 3 above)
Comment (rebeccadee) [reply blogposts/2022/third-party-vulnerabilties-process.md:56]: @jennybergen I think the answer is no, but do you know if we have something to link to here yet?

**Review by nickmyyz — APPROVED** [review]
I read it over again and still think it's ready to go (minus the to-dos of course)! The only thing I might take a look at is whether you could add more paragraph breaks here and there (the intro paragraph for example is a bit long) but I wouldn't consider that blocking.

File: blogposts/2022/third-party-vulnerabilties-process.md (hunk 3 above)
Comment (rebeccadee) [reply blogposts/2022/third-party-vulnerabilties-process.md:56]: I will come back and add this when we have it

**Comment by rebeccadee** [comment]
@nickmyyz / @andreeleuterio I didn't realise that making further changes to the PR would invalidate the previous approval 🙄  Would one of you be able to approve and merge today please? 🙏 I understand why we want to enforce reviews before merging but this is proving to be a bit of a pain!!

**Review by andreeleuterio — DISMISSED** [review]

**Comment by andreeleuterio** [comment]
@rebeccadee I got you

**Review by amieroth — APPROVED** [review]

**Pull request merged**